tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -34,12 +34,14 @@
 # Behave like memcache when it comes to connection errors
 DJANGO_REDIS_IGNORE_EXCEPTIONS = True
 
-# Elasticsearch connection parameters
-ELASTIC_SEARCH_CONFIG = [{
-  {% if ELASTICSEARCH_SCHEME == "https" %}"use_ssl": True,{% endif %}
-  "host": "{{ ELASTICSEARCH_HOST }}",
-  "port": {{ ELASTICSEARCH_PORT }},
-}]
+# Meilisearch connection parameters
```

Comment:
nit: I don't think this comment is needed here, the code ahead is self-explanatory.

Reply:
Sure, it's obvious, but I'd like to keep this comment for proper separation of the different sections.